bootstrap/017_query_history.sql

```diff
@@ -166,3 +170,18 @@ BEGIN
         call internal.set_config(:key, 'true');
     end if;
 END;
+
+create or replace procedure internal.update_qtag_day()
```

Comment:
Also, would be great to have a simple unit test to verify this proc, even if it is naive. We likely wouldn't notice breakages here until the app is deployed, given grants, and finalize_setup is run.

Reply:
added soemthign silly, at least it will check that things work